tox.ini

```diff
@@ -154,3 +154,34 @@ changedir=packages/bootstrap/extra
 commands=
   pip install .
   pytest --basetemp={envtmpdir} {posargs}
+
+
+[testenv:py35-collect-tests]
+platform=linux|darwin
+# See the following link for AWS_* environment variables:
+# http://boto3.readthedocs.io/en/latest/guide/configuration.html#configuring-credentials
+passenv =
+  TEAMCITY_VERSION
+  SSH_AUTH_SOCK
+  AZURE_PROD_STORAGE_ACCOUNT
+  AZURE_PROD_STORAGE_ACCESS_KEY
+  AZURE_DEV_STORAGE_ACCOUNT
+  AZURE_DEV_STORAGE_ACCESS_KEY
+  AWS_ACCESS_KEY_ID
+  AWS_SECRET_ACCESS_KEY
+  AWS_SESSION_TOKEN
+  AWS_PROFILE
+  AWS_CONFIG_FILE
+  AWS_DEFAULT_REGION
+deps =
```

Comment:
Let's be careful - we are duplicating dependencies - anyone adding some in one place should know to add them in another.

Perhaps this should be a `requirements.txt` file?

Reply:
Possibly, yeah.. but I ran into this today: https://github.com/tox-dev/tox/issues/149

Reply:
Maybe https://github.com/signalpillar/tox-battery would help?

Reply:
It seems like you have to install that dependency before you run tox :/

Reply:
Is that a big problem? Is that worse than not having a `requirements.txt` (and therefore not really helping folks who are not using `tox`)?

Reply:
It might not be a problem, but we've already got a ton of dependencies to install before you can get started: https://github.com/dcos/dcos/tree/dfb9df89f0863d9423bf814578c7c9ceb05afb30#development-environment

I'm not keen on adding more, but I don't feel super strongly about it. Maybe we could patch upstream tox.